### PR TITLE
OETH N-01 - Scope

### PR DESCRIPTION
**Pull request by sparrowDom**
Add scope to variables

### CODE REVIEW

File: contracts/contracts/vault/VaultCore.sol [hunk 1]
```diff
@@ -27,9 +27,9 @@ contract VaultCore is VaultStorage {
     using StableMath for uint256;
     using SafeMath for uint256;
     // max signed int
-    uint256 constant MAX_INT = 2**255 - 1;
+    uint256 private constant MAX_INT = 2**255 - 1;
```
Comment (DanielVF) [inline contracts/contracts/vault/VaultCore.sol:30]: Why are these two private and the others internal?

Comment (sparrowDom) [reply contracts/contracts/vault/VaultCore.sol:30]: My thinking is no Contract is going to be extending the VaultCore.

Comment (DanielVF) [reply contracts/contracts/vault/VaultCore.sol:30]: I'd rather just stick with internal, it shouldn't matter gas or contract size wise.

Comment (sparrowDom) [reply contracts/contracts/vault/VaultCore.sol:30]: ok sounds good

**Pull request merged**